prov/efa/src/rdm/rxr_ep.h

```diff
@@ -310,7 +310,9 @@ static inline uint64_t rxr_get_host_id(char *host_id_file)
 		goto out;
 	}
 
-	fseek(fp, HOST_ID_PREFIX_LENGTH, SEEK_SET);
+	if (fseek(fp, HOST_ID_PREFIX_LENGTH, SEEK_SET) < 0) {
+		EFA_WARN(FI_LOG_EP_CTRL, "Cannot locate host id in file\n");
```

Comment:
should we `goto out` here?

Reply:
Yes. Thanks for catching. My IDE had a glitch.